src/gui/tableeditor/qgstableeditordialog.h

```diff
@@ -156,6 +157,18 @@ class GUI_EXPORT QgsTableEditorDialog : public QMainWindow, private Ui::QgsTable
      */
     void setLayer( QgsMapLayer *layer );
 
+    /**
+     * Returns the manual table associated with the editor.
+     * \since QGIS 3.42
+     */
+    QgsLayoutItemManualTable *table() const;
+
+    /**
+     * Sets the \a table associated with the editor.
+     * \since QGIS 3.42
+     */
+    void setTable( QgsLayoutItemManualTable *table );
```

Comment:
Let's SIP_SKIP these too, there's no need for them to be public API:


```suggestion
    QgsLayoutItemManualTable *table() const SIP_SKIP;

    /**
     * Sets the \a table associated with the editor.
     * \since QGIS 3.42
     */
    void setTable( QgsLayoutItemManualTable *table ) SIP_SKIP;
```